src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -1168,24 +1168,25 @@ public void cannotCreateRecallRequestWhenItemIsMissing()
   }
 
   @Test
-  public void cannotCreateRecallRequestWhenItemIsPaged()
+  public void canCreateRecallRequestWhenItemIsPaged()
     throws InterruptedException,
     ExecutionException,
     TimeoutException,
     MalformedURLException {
 
     final IndividualResource requestPickupServicePoint = servicePointsFixture.cd1();
-    final IndividualResource pagedItem = setupPagedItem(requestPickupServicePoint, itemsFixture, requestsClient, usersFixture);
-
+    final IndividualResource smallAngryPlannet = setupPagedItem(requestPickupServicePoint, itemsFixture, requestsClient, usersFixture);
+    final IndividualResource pagedItem = itemsClient.get(smallAngryPlannet);
+    
     final Response recallResponse = requestsClient.attemptCreate(new RequestBuilder()
       .recall()
       .forItem(pagedItem)
       .withPickupServicePointId(requestPickupServicePoint.getId())
       .by(usersFixture.jessica()));
 
-    assertThat(recallResponse.getJson(), hasErrorWith(allOf(
-      hasMessage("Recall requests are not allowed for this patron and item combination"),
-      hasParameter("requestType", "Recall"))));
+      assertThat(recallResponse.getJson().getString("requestType"), is(RequestType.RECALL.getValue()));
+      assertThat(pagedItem.getResponse().getJson().getJsonObject("status").getString("name"), is(ItemStatus.PAGED.getValue()));
```

Comment:
I typically statically import constants / enum values, I find it improves the readability of tests

Reply:
There are several references to these enums throughout. Would you suggest changing them all or only the ones that I added?

Reply:
Personally, I'd use the static import for all of them (IntelliJ just does this when applying that refactoring).

I likely missed them in earlier reviews or have changed my stylistic opinions :-/